docs/notebooks/feasible_sets.pct.py

```diff
@@ -230,6 +240,77 @@ def objective_function(x):
 )
 
 
+# %% [markdown]
+# ## Active learning with Integrated Variance Reduction acquisition function
+#
+# An alternative to the `ExpectedFeasibility` function is called `IntegratedVarianceReduction`. This acquisition has the advantage of allowing batches, as we illustrate below. It is more expensive to compute than `ExpectedFeasibility`, as it writes as an integral over a set of input locations.
+
+# %%
+from trieste.acquisition.function import IntegratedVarianceReduction
+
+# Choose integration points uniformly over the design space
+integration_points = search_space.sample_halton(1000)
+acq_ivr = IntegratedVarianceReduction(
+    integration_points=integration_points,
+    threshold=[
+        threshold,
+    ],
+)
+
+# Set a batch size greater than 1
+rule_ivr = EfficientGlobalOptimization(builder=acq_ivr, num_query_points=3)
+bo = trieste.bayesian_optimizer.BayesianOptimizer(observer, search_space)
+
+num_steps = 10
+result_ivr = bo.optimize(num_steps, initial_data, initial_model, rule_ivr)
+
+final_model_ivr = result_ivr.try_get_final_model()
+dataset_ivr = result_ivr.try_get_final_dataset()
+query_points_ivr = dataset_ivr.query_points.numpy()
+
+plot_excursion_probability(
+    "Final probability of excursion", final_model_ivr, query_points_ivr
+)
+
+# %% [markdown]
+# One can also specify a range of thresholds rather than a single value. The resulting query points are likely to be more speard out than previously.
+thresholds = [50.0, 110.0]
+acq_range = IntegratedVarianceReduction(
+    integration_points=integration_points, threshold=thresholds
+)
+rule_range = EfficientGlobalOptimization(builder=acq_range, num_query_points=3)  # type: ignore
+
+result_range = bo.optimize(num_steps, initial_data, initial_model, rule_range)
+
+# %% [markdown]
+# We can finally draw the probability that the GP value belongs to an interval rather than the probability that it exceeds a single value. We also compare to the probability obtained when optimising for a single value at the center of the range. As expected, the `IntegratedVarianceReduction` spreads query points a bit more, which leads to a sharper probability.
```

Comment:
```suggestion
# We can now illustrate the probability that a point in the search space belongs to the threshold interval rather than the probability that points exceed a single threshold. We compare probability maps obtained with the `IntegratedVarianceReduction` (IVR) when optimising for the threshold range and for the single threshold at the center of the range, as well as to a probability map for the `ExpectedFeasibility` function obtained with a single threshold. As expected, the `IntegratedVarianceReduction` with threshold range spreads query points a bit more, which leads to a sharper probability boundary.
```